modules/README.md

```diff
@@ -0,0 +1,29 @@
+# Modules
+
+## Overview
+
+Modules are the next generation of components in Kyma that are available for local and cluster installation.
+
+Modules are no longer represented by a single helm-chart, but instead are bundled and released within channels through a `ModuleTemplate`, a unique link of a module, and it's desired state of charts and configuration, and a channel.
+
+This directory includes public `ModuleTemplates` for all channels available for consumption within kyma, wherein each channel is represented by a dedicated kustomization.
+
+## Development
+
+To apply all `ModuleTemplates` for your cluster, use the kustomization available in this folder:
+
+```
+kubectl apply -k modules
+```
+
+you will then be able to enable available either with support of Busola through the `Kyma` Custom Resource, or you can manually edit the `Kyma CR` for a cluster with the new Module.
```

Comment:
I think something is missing in this sentence after "available". Do you mean "the available resources" or something else?

Reply:
Agree, the term `modules` is missing